### PR TITLE
Linda lejiete

### PR DESCRIPTION
**Pull request by lindalejiete**
1. Updated html, section-destinations filled with content;
2. Stylesheet updated accordingly;
3. Class names changed from camelCase to kebab-case

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,46 +11,56 @@
...
+            </div>
+            <div class="destinations-moto">
+                <p>It's not about destination,<br> it's about journey</p>
+            </div>
+        </div>
+    </section>
+    <section id="tips" class="section-tips">
+      <div class="tips-text text-dish">
         <h1>Tools</h1>
         <p>Care about product's ingredients, the way products are produced, and the manner they are used. In order to find best eco-friendly, travel-size products, select products which meet rigorous criteria and create one of a kind self-care travel kits. Eco-friendly toothbrushes are made from sustainably sourced materials.</p>
```
Comment (KristianaTR) [inline index.html:28]: code formatting

File: index.html [hunk 2]
```diff
@@ -11,46 +11,56 @@
...
     <section id="home"></section>
     <section id="benefits"></section>
-    <section id="destinations"></section>
-    <section id="tips">
-      <div class="text textDish">
+    <section id="destinations" class="section-destinations">
+        <div class="destinations-wrapper">
+            <div class="destinations-textbox">
+                <h1>How to choose destination?</h1>
+                <p><strong>Ecotourism</strong> is a form of <strong>traveling to natural areas</strong> that involve choosing only sustainable transportation, conserving the environment, and helping the local people while staying respectful to their culture, habits, and tradition. Eco-tourists might aim to educate themselves, <strong>contribute to ecological conservation and the protection of historical sites, and/or benefit local communities</strong> in various ways (e.g., supporting political empowerment, fueling the economy, raising awareness regarding human rights).</p>
```
Comment (KristianaTR) [inline index.html:18]: Code formatting

File: index.html [hunk 3]
```diff
@@ -11,46 +11,56 @@
...
           <input type="email" id="email" name="email" />
         </div>
-        <div class="footerNavigation">
+        <div class="footer-navigation">
           <h1>Site</h1>
-          <p><a href="">Home</a></p>
-          <p><a href="">Benefits</a></p>
-          <p><a href="">Destinations</a></p>
-          <p><a href="">Tips</a></p>
+          <a href=""><p>Home</p></a>
```
Comment (KristianaTR) [inline index.html]: I think that "p" tag here is not needed

File: styles.css [hunk 4]
```diff
@@ -35,47 +35,89 @@ body {
     font-family: var(--font-family-accent);
 }
 
-#destinations,
-#tips {
+.section-destinations,
```
Comment (KristianaTR) [inline styles.css:38]: I suggest styling all sections at the same time. This is what I have in my code:
 section {
    height: 80vh;
    box-sizing: border-box;
    max-width: var(--max-width-desktop);
}
section:nth-child(2n) {
    background-color: var(--color-isabelline);
}

File: index.html (hunk 3 above)
Comment (lindalejiete) [reply index.html]: I added <p> tag, so I can format all <p>-s the same, but we can remove <ps>-s, and in stylesheet I will format <a>-s

File: styles.css (hunk 4 above)
Comment (lindalejiete) [reply styles.css:38]: I agree, except Home section should be like this:

section:first-child {
height: 100vh;
}

File: index.html (hunk 2 above)
Comment (lindalejiete) [reply index.html:18]: Will do it on next pull request

File: index.html (hunk 3 above)
Comment (lindalejiete) [reply index.html]: Also <p> is block element, but <a> inline element. If i remove <p>-s then I have to insert extra styling

**Review by KristianaTR — APPROVED** [review]

**Pull request merged**